### DISCUSSION TITLE
Práctica Vue: Componentes para combinar

### DISCUSSION THREAD

**Discussion by BattiestWinter** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#507] [Q&A]
Hola, mi duda es respecto a que se refiere la práctica con los componentes para combinar sabores y adornos. Tengo entendido que esa acción de "combinar" elementos se hacía en el formulario

**Comment by doguedogue** [comment]
Cada elemento de tu página de la pastelería se va a convertir en un componente:
Encabezado
Barra de navegación 
Footer
Formulario para escoger sabores y toppings
Página de pastelero
Etc.

En la anterior los datos eran fijos, aquí los datos que escojas en un formulario "viajan" a otras páginas/componentes.

🚀

**Reply by RodolfoMorquecho** [reply]
Ya terminaste la practica de Vue? creo que tienes muy bien definido todo y muchos estamos perdidos.
